### PR TITLE
status: add config info in the status output

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
status: add config info in the status output

We are now adding config info in the status output. This will be reflected whenener the user runs ua status with the --format flag

## Test Steps
Launch a machine and check if the output of `ua status --format json` has the config keys. Also, check if changing the path to load the config file from will reflect in ua status

## Desired commit type::

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [x] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: uaclient/config.py
Comment (orndorffgrant) [inline uaclient/config.py:796]: putting the config_path in here feels a little weird. If we have time, could we refactor slightly so config path is determined up in __init__ and set as an attribute on the UAConfig instance rather than in the cfg dict? Just an idea, feel free to disagree

Comment (lucasmoura) [reply uaclient/config.py:796]: I don't think it is that weird since the `config_path` can in fact be configured from the outside. However, I can do the refactoring if needed. @blackboxsw do you share @orndorffgrant feeling that this variable should not be include in the `config` dict ?

Comment (orndorffgrant) [reply uaclient/config.py:820]: True that it can be configured from the outside. The slight difference is that the rest of the contents of `cfg` are configurable via uaclient.conf, while `config_path` isn't.

I agree that it is not that weird though, lets not hold up this PR on this.

File: uaclient/status.py
Comment (orndorffgrant) [inline uaclient/status.py:693]: I agree with the more "functional" style of creating a new dict based on the old one, rather than doing a full copy and mutating the copy. +1

> Random aside: sometimes dict/list comprehensions are hard to read, but this is one of those cases where it perfectly expresses a map, filter, and reduce all in one succinct line. Love it

My only thought to consider is that we're currently passing `cfg` to this function to do this here, but we construct most of the data in `config.py` where we already have `cfg` available. What are your thoughts on adding the config data here vs in `UAConfig._get_config_status()`?

Comment (lucasmoura) [reply uaclient/status.py:693]: That's a good idea, I will refactor the code to use it

**Comment by lucasmoura** [comment]
@orndorffgrant code updated

**Review by orndorffgrant — APPROVED** [review]
Thanks for the updates! This looks great and worked for me. Lets just let Jenkins approve before merging

**Comment by lucasmoura** [comment]
I could not reproduce that failing test locally. I will merge this PR as is and take a look on the daily CI

**Pull request merged**
